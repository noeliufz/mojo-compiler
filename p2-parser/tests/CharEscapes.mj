const
  ALERT = '\a',
  BS = '\b',
  FF = '\f',
  NL = '\n',
  CR = '\r',
  HT = '\t',
  VT = '\v',
  nul = '\000',
  soh = '\001',
  stx = '\002',
  del = '\177',
  pound = 'Â£';
