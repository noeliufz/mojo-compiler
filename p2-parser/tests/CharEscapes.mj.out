const ALERT = '\a' (* 0x7 *);
const BS = '\b' (* 0x8 *);
const FF = '\f' (* 0xc *);
const NL = '\n' (* 0xa *);
const CR = '\r' (* 0xd *);
const HT = '\t' (* 0x9 *);
const VT = '\v' (* 0xb *);
const nul = '\000' (* 0x0 *);
const soh = '\001' (* 0x1 *);
const stx = '\002' (* 0x2 *);
const del = '\177' (* 0x7f *);
const pound = 'Â£' (* 0xa3 *);
